doc/ctsm-docs_container/README.md

```diff
@@ -0,0 +1,53 @@
+# The ctsm-docs container
+This directory and its Dockerfile are used to build a Docker container for building the CTSM documentation. Unless you're a developer working on the container, you probably don't need to care about anything in here.
+
+## Building
+
+If you actually want to build the container, make sure Docker is running. In the Docker Desktop settings, make sure you've enabled the [`continerd` image store](https://docs.docker.com/desktop/features/containerd/), which allows multi-platform builds. Then do:
+```shell
+docker buildx build --platform linux/amd64,linux/arm64 -t ghcr.io/escomp/ctsm/ctsm-docs .
+```
+
+To use your new version for local testing, you'll need to tell doc-builder to use that image. Call `docker images`, which should return something like this:
```

Comment:
I'm not quite sure when you change this. 

The doc_builder points to a fixed tag in build_commands.py for default_docker_image. So whenever developers update the tag for the image it needs to be updated there. And here you change it locally when you want to use your new image you created.

I think it would be good to have a little more explaination on both of those workflows.

Reply:
Although actually that's part of what's below in

Updating doc-builder

Reply:
Maybe what this needs is an explanation that goes through the big picture steps first, so it's easier to see the general flow.